### ISSUE TITLE
[EN][Talent][Mage] Combustion does less damage than expected

### ISSUE THREAD

**Issue by Crusaderz** [Atlantiss/BugTracker#6695]
According to http://www.wowhead.com/spell=11129/combustion#comments :
Here's how this spell works in 4.0.1 and beyond:

1. You cast Combustion on an enemy target.
2. It deals a small, insignificant amount of fire damage on its own.
3. Then the game adds up the total DPS of all the fire dots you have on the target.
4. A new dot, called Combustion, is placed on the target that ticks for that total dps, every second for 10 seconds.

So let's say the following dots are on your target when you cast Combustion...

Ignite ticking for 3000 every 2 seconds (1500 dps)
Pyroblast ticking for 900 every 3 seconds (300 dps)
Living Bomb ticking for 1500 every 3 seconds (500 dps)
Combustion will tick for 2300 every second for 10 seconds, for a total of 23000 damage. Combustion's dot ticks are capable of critting and you also get more ticks the more haste you have. You can also spread Combustion to all targets in about 10 yards with Impact procs for some disgusting AoE damage.

Screenshot:

ignite: 4297 / 2 = 2 148
living bomb: 3403 / 3 = 1 134
pyro dot: 2104 / 3 = 701

Expected damage = 3 983. Real damage: 3 787
Training dummy tests also show numbers less than expected

**Comment by Kejmil** [comment]
pyro dot and living bomb dot are not taken dirrectly of the target but theirs value for combustion are "how much dps would we do if we were cast on target in the moment when fire mage click combustion"

http://forums.elitistjerks.com/forums/topic/128507-433-fire-mage-compendium/

"The power of the living bomb DOT and the pyro DOT on the hand are much complicated to calculate. Keep in mind that Combustion only check for the presence of the LB and pyro dot as it will recalculate their power with your current stats at the time of casting Combustion, not the ones you had when casting the DOTs."

also don't ever take into account tooltip dmg of combustion on ats, it's f*cked up

**Comment by Crusaderz** [comment]
Temporary buffs is clearly not the case. I'm level 84 without any "proc on hit" trinkets so my spellpower is always same and my dots do same damage before and after combustion.

**Comment by aneq** [comment]
yes, but you do have molten fury which increases your damage done to targets below 35% health
You're not even lvl 85 so I doubt you used a lvl 88 heroic training dummy which has 100 mln hp, you would miss way to often

**Comment by Crusaderz** [comment]
My combustion doing expected damage when i'm at 0\3 molten fury. But when i spec into it, combustion again doing less damage.

![combust](https://cloud.githubusercontent.com/assets/3641625/13528830/a07aea20-e229-11e5-8703-fbfeb0ae7af4.PNG)

living bomb 3425 / 3 = 1 141
ignie: 5759 / 2 = 2 879
pyro dot: 2121 / 3 = 707
expected damage: 4 727, real: 4529

Dummy is under 35% hp (1hp) so molten fury should kick in and seems like it's not.

**Comment by Crusaderz** [comment]
http://www.mmo-champion.com/threads/1062466-Returning-Fire-DPS-questions

Your ilvl is about the same as my alt mage, and I pulled about 31k on the last LFR Ultraxion, so on normal with Fading Light you should be able to pull at least 30k. First thing I notice is your haste. You should aim for 1505 haste to get the extra tick of combustion (with 2pc T13 fully stacked, 2005 haste without the 2pc). With your gear it should be pretty easy with some reforging. Also, you only used combustion twice on your Ultraxion fight - you need to get 3 off. The decision of whether or not to use combustion at any given moment is tough, and you'll get better at it with experience. **The 3rd combustion is pretty important in this case because it would most likely be cast during Molten Fury**. Getting the most out of combustion will probably get you over 30k.

**Comment by BialyADI** [comment]
http://www.mmo-champion.com/threads/1061624-How-does-Combustion-lt-really-gt-calculate-its-damage
His Combustion Should hit for 10664 Dmg Tick but it hit for 10334 that how it work.

**Comment by Zhalox123** [comment]
Combustion is still doing too low dmg

**Comment by walkthedog** [comment]
"The power of the living bomb DOT and the pyro DOT on the hand are much complicated to calculate. Keep in mind that Combustion only check for the presence of the LB and pyro dot as it will recalculate their power with your current stats at the time of casting Combustion, not the ones you had when casting the DOTs."

I might be missing something so correct me if I'm wrong, but it doesn't seem to be this way here. Combustion appears to be calculated based on the power of the Living Bomb that is present on the target instead of its potential power at the moment of casting Combustion. I assume the same goes for Pyro dot. Combustion used with a living bomb on the target will tick for the same amount regardless of your stats at the moment you use it.
